dashboard/app/views/maker/display_code.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1= 'Login to the Maker App with Google'
+
+%p Please copy and paste this code into the Maker App to login to Code.org
+
+%input#maker-code{:type=>"text", :readonly=>true, :value=>@secret_code}
```

Comment:
nit: let's make this input and the button below railsy with [text_field_tag](https://apidock.com/rails/ActionView/Helpers/FormTagHelper/text_field_tag) (which accepts a [readonly option](https://coderwall.com/p/r2cqhg/don-t-confuse-disabled-input-option-with-readonly-one)) and [button_tag](https://apidock.com/rails/ActionView/Helpers/FormTagHelper/button_tag)